terraform/gcp/security/gcp-gke-client-certificate-disabled.yaml

```diff
@@ -0,0 +1,34 @@
+rules:
+- id: gcp-gke-client-certificate-disabled
+  patterns:
+    - pattern: resource
+    - pattern-inside: |
+        resource "google_container_cluster" "..." {
+        ...
+        }
+    - pattern-not-inside: |
+        resource "google_container_cluster" "..." {
+        ...
+        master_auth {
+          ...
+          client_certificate_config {
+            ...
+            issue_client_certificate = true

```

Comment:
The message below says to ensure that client certificate authentication is disabled. This `pattern-not-inside` filters out resources where `issue_client_certificate` is set to `true`. Is this the opposite of what's supposed to be happening?

Reply:
That was supposed to be the opposite (https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#restrict_authn_methods)